package.json

```diff
@@ -35,32 +35,28 @@
         "@fluid-project/eleventy-plugin-fluid": "github:fluid-project/eleventy-plugin-fluid#ee75b78"
     },
     "devDependencies": {
-        "@babel/cli": "7.8.4",
-        "@babel/core": "7.9.6",
-        "@babel/preset-env": "7.9.6",
         "@commitlint/cli": "11.0.0",
         "@commitlint/config-conventional": "11.0.0",
-        "babel-loader": "8.1.0",
-        "babel-plugin-transform-es2015-modules-commonjs-simple": "6.7.4",
         "cross-env": "7.0.2",
         "debug": "4.2.0",
-        "file-loader": "6.0.0",
-        "fluid-grunt-lint-all": "1.0.8",
+        "fast-glob": "3.2.4",
+        "fluid-grunt-lint-all": "github:jobara/fluid-grunt-lint-all#issue-14",
         "grunt": "1.3.0",
-        "grunt-contrib-clean": "2.0.0",
-        "grunt-contrib-copy": "1.0.0",
         "html-minifier": "4.0.0",
-        "html-webpack-plugin": "4.5.0",
         "husky": "4.3.0",
         "image-size": "0.9.1",
         "infusion": "3.0.0-dev.20200326T173810Z.24ddb2718",
         "jsdom": "16.4.0",
+        "laravel-mix": "5.0.7",
+        "move-file": "2.0.0",
         "npm-run-all": "4.1.5",
+        "resolve-url-loader": "3.1.1",
```

Comment:
`resolve-url-loader`, `sass-loader` and `stylus-loader` are all listed as dependencies but are not used. Should they be added to the webpack configuration to provide an example or a baseline config? Would this be unnecessary due to the way Laravel Mix does its thing?

Reply:
They're used internally by Laravel Mix so they don't need to be imported or used in an example config in any way.